plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -133,7 +133,10 @@ public static function get_tracking_data() {
 		$data = array();
 
 		// General site info.
-		$data['url'] = home_url();
+		$data['url']      = home_url();
+		$data['store_id'] = get_option( 'woocommerce_store_id', null );
+		$data['blog_id']  = class_exists( 'Jetpack_Options' ) ? Jetpack_Options::get_option( 'id' ) : null;
```

Comment:
We're adding the Jetpack `blog_id` here to make WC Tracker snapshots consistent with Tracks events.

Reply:
I don't know that for certain. I was following the pattern that was previously established in `WC_Tracker`